benchmarks/src/main/java/io/improbable/keanu/benchmarks/BinaryOperation.java

```diff
@@ -34,6 +34,14 @@ public DoubleTensor apply(DoubleTensor lhs, DoubleTensor rhs) {
         public double apply(double lhs, double rhs) {
             return lhs / rhs;
         }
+    },
+    MATRIX_MULTIPLY{
+        public DoubleTensor apply(DoubleTensor lhs, DoubleTensor rhs) {
+            return lhs.matrixMultiply(rhs);
+        }
+        public double apply(double lhs, double rhs) {
+            return lhs / rhs;
```

Comment:
```suggestion
            return lhs * rhs;
```

Reply:
I'm no longer using the primitive op so I'll just remove it.